### ISSUE TITLE
Zensus Data 2011 can't be found

### ISSUE THREAD

**Issue by xsaschako** [DLR-SC/memilio#695]
### Bug description

https://opendata.arcgis.com/datasets/abad92e8eead46a4b0d252ee9438eb53_1.csv
doesn't exist anymore and leads to several test problems

### Version

Any

### To reproduce

Run python data tests, see GH Actions.

### Relevant log output

```zsh
Downloading https://opendata.arcgis.com/datasets/abad92e8eead46a4b0d252ee9438eb53_1.csv 
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/memilio/epidata/getDataIntoPandasDataFrame.py", line 194, in get_file
    file = download_file(
  File "/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/memilio/epidata/getDataIntoPandasDataFrame.py", line 100, in download_file
    raise requests.exceptions.HTTPError("HTTPError: "+str(req.status_code))
requests.exceptions.HTTPError: HTTPError: 404

During handling of the above exception, another exception occurred:
```

### Checklist

- [X] Attached labels, especially loc:: or model:: labels.
- [X] Linked to project

**Comment by patricklnz** [comment]
See #662.
I'm already working on it ;)

**Comment by xsaschako** [comment]
@patricklnz ups, thank you! Let me know when I can review it so we can merge that asap

**Comment by patricklnz** [comment]
Closed by #684.
